### PR TITLE
WIP: Vertical Alignment

### PR DESCRIPTION
**Pull request by Stevoisiak**
To prevent scope creep, I'm tackling this PR one file at a time. I'll squash them once they are reviewed.

### CODE REVIEW

**Comment by BhaaLseN** [comment]
-0 from my side, I don't think vertical alignment inside the methods code is that important (that is, assignments etc.), especially considering the huge amount of work to actually get it done in hindsight (as you are doing now).
For structs, classes, lookup-tables and similar global stuff however, I'm all in favor of getting some structure in them (which most of them already have).

Feel free to ignore my 2ct if everyone else wants that sort of alignment everywhere.

File: Source/Core/DolphinWX/AboutDolphin.cpp [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
 AboutDolphin::AboutDolphin(wxWindow *parent, wxWindowID id,
 		const wxString &title, const wxPoint &position,
-		const wxSize& size, long style)
+		const wxSize   &size,  long style)
```
Comment (RachelBryk) [inline Source/Core/DolphinWX/AboutDolphin.cpp]: keep & against the type

File: Source/Core/DolphinWX/AboutDolphin.h [hunk 2]
```diff
@@ -19,7 +19,7 @@ class AboutDolphin : public wxDialog
 	AboutDolphin(wxWindow *parent,
 		wxWindowID id = wxID_ANY,
 		const wxString &title = _("About Dolphin"),
-		const wxPoint& pos = wxDefaultPosition,
-		const wxSize& size = wxDefaultSize,
+		const wxPoint  &pos   = wxDefaultPosition,
+		const wxSize   &size  = wxDefaultSize,
```
Comment (RachelBryk) [inline Source/Core/DolphinWX/AboutDolphin.h]: & against type here too

File: Source/Core/DolphinWX/Frame.cpp [hunk 3]
```diff
@@ -359,14 +359,14 @@ bool CFrame::InitControllers()
 #if defined(HAVE_X11) && HAVE_X11
 		Window win = X11Utils::XWindowFromHandle(GetHandle());
 		HotkeyManagerEmu::Initialize(reinterpret_cast<void*>(win));
-		Pad::Initialize(reinterpret_cast<void*>(win));
-		Keyboard::Initialize(reinterpret_cast<void*>(win));
-		Wiimote::Initialize(reinterpret_cast<void*>(win));
+		Pad::Initialize             (reinterpret_cast<void*>(win));
+		Keyboard::Initialize        (reinterpret_cast<void*>(win));
+		Wiimote::Initialize         (reinterpret_cast<void*>(win));
```
Comment (lioncash) [inline Source/Core/DolphinWX/Frame.cpp]: please don't do this sort of thing with functions, leave the actual argument lists against the call.

**Comment by lioncash** [comment]
I can get vertically aligning stuff when there's a large list of things that look better in a tabular format. However, for most of this, it's really just creating more line noise when reading the code.

**Comment by DZCreeper** [comment]
I fail to see how this makes code more readable.

**Pull request closed without merging**